app/en/home/hosting-overview/page.mdx

```diff
@@ -5,24 +5,23 @@ description: "Learn about the different ways to host Arcade"
 
 # Hosting Options
 
-The best way to use Arcade is to use our cloud service.  However, if you need to run Arcade locally to either connect your tools to local-only resources (e.g. a local database or filesystem), or your policies require it, Arcade has you covered!
+The easiest and best way to use Arcade is to use our cloud service - sign up for free at [https://api.arcade.dev](https://api.arcade.dev).  However, you might need to connect your tools to local resources (e.g. a local database or filesystem) or keep data within your own infrastructure.  Don't worry, Arcade has you covered!
 
-## Customizing Auth
+## MCP Server Deployment
 
-You don't have to host Arcade to customize your auth experiences. Arcade's cloud service supports a number of auth providers out of the box, but you can always provide your own OAuth app credentials. We recommend doing this for any production use so that you can have isolated rate limits with the OAuth service provider and give your users a consistent experience when they go through an auth flow.
+You can route and manage tool calls from your agents to MCP servers hosted anywhere - on your machine, on your own infrastructure, in a private cloud, or on our cloud service.  This allows you to mix the best public tools with your own private tools.
 
-You can still use the same tools when you customize your auth, no code changes are required.
+See [On-premise MCP Servers](/home/deployment/on-prem-mcp) for more information about how to use your own MCP servers in a federated deployment, and see [Arcade Deploy](/home/serve-tools/arcade-deploy) to learn how to deploy to Arcade's cloud.
 
-See [Customizing Auth](/home/auth-providers) for more information.
+## Customizing Auth
 
-## Hybrid Deployments
+You don't have to host Arcade to customize your auth experiences. Arcade's cloud service supports a number of auth providers out of the box, but you can always provide your own OAuth app credentials. We recommend doing this for any production use-case so that you can have isolated rate limits with the OAuth service provider and give your users a consistent experience when they go through an auth flow.
```

Comment:
```suggestion
You don't have to self-host Arcade to customize your auth experiences. Arcade's cloud service supports a number of auth providers out of the box, and you can provide your own OAuth app credentials to brand your end-user experience. We recommend doing this for all production use cases, so that you can have isolated rate limits with the OAuth service provider and you can give your users a consistent experience when they go through an auth flow.
```